Makefile

```diff
@@ -32,10 +33,15 @@ cross: clean
 .PHONY: vendor
 vendor: vendor.conf
 	@vndr 2> /dev/null
-	@script/validate/check-git-diff vendor
+	@scripts/validate/check-git-diff vendor
+
+## Generate man pages from go source and markdown
+.PHONY: manpages
+manpages:
+	@man/generate.sh
```

Comment:
Not entirely sure how `make` deals with targets or dependencies which are directories rather then files, but if that sort of thing works (I guess it just `stat`s and uses the `mtime` just like for a file), sure.